python/lsst/pipe/tasks/insertFakes.py

```diff
@@ -41,6 +41,110 @@
 __all__ = ["InsertFakesConfig", "InsertFakesTask"]
 
 
+def _add_fake_sources(image, objects, calibFluxRadius=12.0, logger=None):
+    """Add fake sources to the given image
+
+    Parameters
+    ----------
+    image : `lsst.afw.image.exposure.exposure.ExposureF`
```

Comment:
Looking elsewhere in the stack (e.g., in [meas_algorithms](https://github.com/lsst/meas_algorithms/blob/6590b197636fb1452f56009ad2efc69934c71ecb/python/lsst/meas/algorithms/detection.py#L211)), I think that an exposure type normally seems to be referred to as `exposure`, or similar. I worry that `image` is a little confusing, especially as `image.image` is called elsewhere in the script.

Reply:
I agree.  I'll switch this one to `exposure` and other similar cases that are newly introduced in this ticket.  For now I'll leave `image` where it's similarly used in the pre-existing methods though.